### PR TITLE
spec: Update fundamentals.

### PR DESCRIPTION
**Pull request by JoeGruffins**
Update the config response and replace fees with bonds.

part of #1641

Can be seen at https://github.com/JoeGruffins/dcrdex/blob/updatingspecstuff/spec/fundamentals.mediawiki

### CODE REVIEW

File: spec/fundamentals.mediawiki [hunk 1]
```diff
@@ -15,75 +37,120 @@ This section describes each of these design aspects.
...
 
-Every asset is assigned a unique integer ID that will be used to identify the
-asset in serialized structures. Whenever possible, the ID is the same as the
-BIP-0044 registered coin type index
-[https://github.com/satoshilabs/slips/blob/master/slip-0044.md &#91;6&#93;].
-
-When the asset is the quote asset, a '''price increment''' is enforced.
-The '''rate''', '''''r''''', of a limit order must be an integer multiple of the
-''price increment'', '''''p'''''.
+For the quote asset, a '''rate step''' (<code>ratestep</code>) is enforced. The ''rate''
```
Comment (martonp) [inline spec/fundamentals.mediawiki:120]: You could write: The "rate" is in units of the quote asset, and in must be specified in integer multiples of the "rate step".

File: spec/fundamentals.mediawiki [hunk 2]
```diff
@@ -15,75 +37,120 @@ This section describes each of these design aspects.
...
+broadcast, as well as on the second swap transaction, before the first
+redemption is broadcast.
+
+The '''max fee rate''' (<code>maxfeerate</code>) is the greatest fee the server
+may expect the client to pay on initiation transactions.
+
+The '''unit info''' (<code>unitinfo</code>) contains information on unit names
+and includes the following variables:
+
+* The '''atomic unit''' (<code>atomicUnit</code>) is the smallest unit the asset package will use in communications. i.e. Sats for btc and gwei for eth.
```
[the remark this answers is not in the document]
Comment (JoeGruffins) [reply spec/fundamentals.mediawiki:95]: not the "smallest unit", but "the smallest unit the asset package will use in communications"

I wanted to convey that it is the base unit but may not be the actual base unit, like gwei. I'll try to be more clear.

[the remark this answers is not in the document]
Comment (JoeGruffins) [reply spec/fundamentals.mediawiki:95]: Actually, I guess yours if good, it's obviously gwei.

[the remark this answers is not in the document]
Comment (JoeGruffins) [reply spec/fundamentals.mediawiki:95]: Changed.

File: spec/fundamentals.mediawiki [hunk 3]
```diff
@@ -15,75 +37,120 @@ This section describes each of these design aspects.
...
+
+The '''version''' (<code>version</code>) is the bond version. A higher version indicates updated bond construction.
+
+The '''id''' (<code>id</code>) is the bond asset's BIP-0044 registered [https://github.com/satoshilabs/slips/blob/master/slip-0044.md coin type index]
+[[references.mediawiki|<nowiki>[6]</nowiki>]]
+
+The '''confirmations''' (<code>confs</code>) are the confirmations needed in order for the bond to be
+considered valid.
+
+The '''amount''' (<code>amount</code>) is the amount of the lowest communicatable value of the asset
```
[the remark this answers is not in the document]
Comment (JoeGruffins) [reply spec/fundamentals.mediawiki:152]: Changed to
```
The '''amount''' (<code>amount</code>) is how much is required for one bond, and
the amount posted must be divisible by this amount. The amount will be in the
asset's [[#asset-variable|asset's]] <code>atomic unit</code>.
```

File: spec/fundamentals.mediawiki [hunk 4]
```diff
@@ -15,75 +37,120 @@ This section describes each of these design aspects.
...
+asset in serialized structures. Whenever possible, the ID is the same as the
+BIP-0044 registered [https://github.com/satoshilabs/slips/blob/master/slip-0044.md coin type index]
+[[references.mediawiki|<nowiki>[6]</nowiki>]]. The '''symbol''' (<code>symbol</code>) is also found there.
+
+The '''swap confirmations''' (<code>swapconf</code>) is the number of confirmations required during
+settlement on the first swap transaction, before the second swap transaction is
+broadcast, as well as on the second swap transaction, before the first
+redemption is broadcast.
+
+The '''max fee rate''' (<code>maxfeerate</code>) is the greatest fee the server
```
[the remark this answers is not in the document]
Comment (JoeGruffins) [reply spec/fundamentals.mediawiki:89]: Added.

File: spec/fundamentals.mediawiki (hunk 1 above)
Comment (JoeGruffins) [reply spec/fundamentals.mediawiki:120]: Changed to:
```
A trade's rate ''r'' is in units of the quote asset, and must be specified in
integer multiples of the '''rate step''' (<code>ratestep</code>) shown here as ''p''.
```

**Comment by JoeGruffins** [comment]
Hopefully fixing @martonp comments https://github.com/decred/dcrdex/compare/d4f5d11401319cbae226b5bc2c85d78f83b94526..cf4a1546a83b33ec28297787a957c316abd18b23

**Review by chappjc — APPROVED** [review]

**Pull request merged**